components/measurement/DetailsHeader.js

```diff
@@ -43,7 +44,13 @@ const DetailsHeader = ({testName, runtime, notice}) => {
           />
         </Box>
         <Box ml={2}>
-          <Text fontSize={20}>{metadata.name}</Text>
+          <Link href={metadata.info} target='_blank'>
```

Comment:
I now realised that the behaviour is that of opening it in a new window.

I don't think we should be doing that. If the user wants to open it in a new window they can alt-click on the link.

Reply:
Opened this PR for it: https://github.com/ooni/explorer/pull/471